ci/release-info-v2.py

```diff
@@ -0,0 +1,96 @@
+import json
+import os
+import re
+import sys
+from datetime import date
+from subprocess import Popen, PIPE
+
+def read_version(crate):
+    # Reads package version from workspace manifest file
+    channel = Popen(
+        ['cargo', 'read-manifest', '--manifest-path', './{}/Cargo.toml'.format(crate)],
+        stdout=PIPE
+    )
+
+    # parse json data
+    data = json.load(channel.stdout)
+    version = data['version']
+    return version
+
+def check_version(ver):
+    # Checks package version for matching with the current tag reference
+    if ref is not None and ref != 'refs/tags/v' + str(ver):
```

Comment:
This line is **very** confusing since `ref` is defined somewhere else and not before this function. I would rather write `def check_version(ref, ver)`.

Reply:
ref is defined before this function and passing by reference, passing ref as parameter for this function prevents the use of this function in parallel map, 'cause map stops on most short iterable and ref is a one item container.